apps/prairielearn/src/pages/authLogin/authLogin.html.ts

```diff
@@ -185,3 +186,38 @@ export function AuthLogin({ institutionAuthnProviders, service, resLocals }: Aut
     </html>
   `.toString();
 }
+
+function DevModeBypass() {
+  return html`
+    <a class="btn btn-success w-100" href="/pl/dev_login" role="button">
+      <span class="font-weight-bold">Dev Mode Bypass</span>
+    </a>
+    <small class="text-muted">You will be authenticated as <tt>${config.authUid}</tt>.</small>
```

Comment:
Small enhancement to show which user you'll actually auth as. I opted to leave the "bypass" button as a faster option for folks who just want to auth without filling in a form.